### DISCUSSION TITLE
Text previews

### DISCUSSION THREAD

**Discussion by albheim** [kamiyaa/joshuto#259] [Q&A] [answered]
Should text from "normal" text files just work? They don't seem to do that for me, I only ever get previews on folders.

It is not entirely clear to me what preview I should expect to work by default, since in the list of [features](https://github.com/kamiyaa/joshuto/blob/main/README.md#features) we are told there is file previews, and that we should look at the [image preview instructions](https://github.com/kamiyaa/joshuto/blob/main/docs/image_previews.md) for more information. Though in the image preview instructions I mostly find (unsurprisingly) how to show an image, and a little of how to show image with some text metadata. Though there is also the line
> Keep in mind that the result of the preview script you use for textual previews is cached by Joshuto and not called every time a file is focused, but the “shown” hook is.

which makes me think that maybe I have to add a script for textual previews? How would I go about doing that?

**Comment by daolanfler** [comment, marked as answer]
After I copy config files to ~/. config/joshuto , text files preview just works. Though text highlighting only works for a small set of filetypes
